hack/move-upstream.sh

```diff
@@ -18,6 +18,8 @@ repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
 package="${UPSTREAM_PACKAGE:-pkg/api}"
 
 patch="${TMPDIR}/patch"
+rm -rf "${patch}"
+mkdir -p "${patch}"
 relativedir="../../../${repo}"
```

Comment:
If you don't follow the GOPATH convention we don't like you anyway.  Can't be GOPATH